### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by HITGIF**
- Add gitter chat badge
- Add contents section
- Add references to Eclipse and IntelliJ IDEA
- Update contribution guideline
- Fix grammar errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,110 @@
...
-4. Download and install [Android Studio](https://developer.android.com/studio/index.html), an IDE for android application development.
-5. You will also need to download the Android SDK from the IDE itself.
-6. Import the **forked** repository into Android Studio by clicking on files-->Open and navigate to the directory where you forked the repository.
 
-* If you want to just download the app without contributing, then skip steps 1 & 2 and just download [.zip](https://github.com/jboss-outreach/compressor-head-android/archive/master.zip) file.
+### IDE Setup
+##### It is recommanded to use Android Studio:
+1. Download and install [Android Studio](https://developer.android.com/studio/index.html).
+2. You will also need to download the Android SDK from the IDE itself.
+3. Import the **forked** repository into Android Studio by clicking on `files --> Open` and navigate to the directory where you forked the repository.
```
Comment (neelanshsahai) [inline README.md]: The clone is imported, not the fork :stuck_out_tongue_closed_eyes:

File: README.md [hunk 2]
```diff
@@ -1,59 +1,110 @@
...
-## Contributing to the project
+ - Setup a [Android Virtual Device](https://developer.android.com/studio/run/managing-avds.html) in the IDE.
+ - Click **Run** on the Android Studio tool bar, or **Shift + F10** and then choose the newly created virtual device to [run the app](https://developer.android.com/studio/run/device.html).
+
+## <a id = "contributing"> </a> Contributing to the project
+
+
+**Step 1: Fork this project:**
+
+Go to the top right of the project page and click on "Fork". A clone of this repo will be created on your GitHub account.
```
Comment (neelanshsahai) [inline README.md]: A fork is created *
After forking the next step is cloning ^^
and then comes making branches

File: README.md [hunk 3]
```diff
@@ -1,59 +1,110 @@
...
 
-1. Make your intended changes on your locally cloned repository.
-2. *Commit* the changes and *push* the commit to your forked repository on GitHub.
-3. Ensure the changes on the commit pushed to your GitHub fork are correct.
-4. Create a [*Pull Request*](https://help.github.com/articles/about-pull-requests/) requesting to merge the commits on your fork to this repository.
-5. Write a very conscise but informative pull request message explaining the test cases in your pull request. Remember to use your words wisely. 
+**Step 2: Code your changes**:
 
+Create a new branch, and create/edit files as per your coding requirements. Ensure that your code is clean and efficient, and avoid redundancies. It is also advised to follow naming conventions as and where specified. Also make sure that your code is your own, and is not closed-source or stolen.
```
Comment (neelanshsahai) [inline README.md]: Showing the git command here as well will be a good idea.

**Comment by HITGIF** [comment]
Done =)

**Pull request merged**